### ISSUE TITLE
CrossValidation issue

### ISSUE THREAD

**Issue by manju22412** [haifengl/smile#845]
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 10 out of bounds for length 10
	at smile.validation.metric.CrossEntropy.of(CrossEntropy.java:41)
	at smile.validation.ClassificationMetrics.of(ClassificationMetrics.java:257)
	at smile.validation.ClassificationValidation.<init>(ClassificationValidation.java:86)
	at smile.validation.ClassificationValidation.of(ClassificationValidation.java:179)
	at smile.validation.ClassificationValidation.of(ClassificationValidation.java:204)
	at smile.validation.CrossValidation.stratify(CrossValidation.java:337)

Whenever there’s a unique or low-frequency label in the training data, I encounter an exception for any k-fold value. Why does this happen? Regardless of label counts, it should ideally validate based on the available data and still produce results, right?

**Comment by haifengl** [comment]
It is likely that you have only very few samples for some class. If you can share your data, we can investigate further. Thanks.
